Book.js

```diff
@@ -0,0 +1,44 @@
+const saveToLocalStorage = (key, data) => localStorage.setItem(key, JSON.stringify(data));
+const getFromLocalStorage = (key) => JSON.parse(localStorage.getItem(key));
+
+class Book {
+  constructor(title, author, id = undefined) {
```

Comment:
- [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting numbers as inputs. Kindly endeavor to provide validation for the same